database/migration/src/main/resources/changesets/add_hog_group_in_workflow_store.xml

```diff
@@ -4,7 +4,7 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
 
-    <changeSet id="add_hog_group_in_workflow_store" author="cjllanwarne">
+    <changeSet id="add_hog_group_in_workflow_store" author="cjllanwarne" dbms="hsqldb,mysql,postgresql">
```

Comment:
Why does this changeset have additional `postgresql` while some of the changesets only have `mysql,hsqldb`?

Reply:
![wolfcastle](https://user-images.githubusercontent.com/791985/60941191-36dc6980-a2ac-11e9-8f55-9b868e59d82f.jpg)
That's the ~joke~ PR

We were running into problems on new DBMS such as `postgresql` and `mariadb` because changelog changesets were executing unexpectedly.

So, in addition to other fixes, as of this PR all changesets must explicitly list the DBMS that they support.

This particular changelog came after `postgresql` was introduced earlier, so the changeset has been fixed to include that DBMS. However the column should NOT yet be added to databases identifying themselves as `mariadb`, as the `mariadb` schema initialization exists in the following changelog.